JavaScript/Twitter_Unfollowers/README.md

```diff
@@ -0,0 +1,23 @@
+# Twitter_Unfollowers
```

Comment:
After and before (except the first one) every heading please introduce a blank line.